style-dictionary/utils/token-names.ts

```diff
@@ -630,6 +631,10 @@ export type SpacingTokenName =
   | 'spaceModalContentBottom'
   | 'spaceModalHorizontal'
   | 'spaceOptionIconBigTop'
+  | 'spacePanelContentBottom'
+  | 'spacePanelContentTop'
+  | 'spacePanelDividerMarginHorizontal'
```

Comment:
Any specific reason why this token contains the word `Margin` and the others don't? Usually we don't include this word in tokens, and it used for paddings too, so could it be `spacePanelDividerHorizontal` instead?